application/src/main/java/be/xplore/pricescraper/schedules/ScrapeItemsJob.java

```diff
@@ -28,6 +28,7 @@ public class ScrapeItemsJob {
    */
   @Scheduled(fixedDelay = 5000)
```

Comment:
this class could be an adapter as then its application can be changed easily, this module should be as dumb as  possible and basically just puzzle together the dependencies so you get a functioning application